README.md

```diff
@@ -4,3 +4,190 @@
 ![Cocoapods platforms](https://img.shields.io/cocoapods/p/CellKit.svg)
 ![License](https://img.shields.io/cocoapods/l/CellKit.svg)
 ![Continuous integration](https://img.shields.io/bitrise/57c06040e101a852.svg?token=xvYC9NW6ZszIzFdjkapmVg)
+
+CellKit is a Swift package that streamlines the workflow with cells in UITableView and UICollectionView. No more registering cell classes or XIBs, no more dequeueing cells and setting its view models. CellKit handles this all.
+
+## Installation
+### Swift Package
+Add following line to your swift package dependencies, or in Xcode, go to `File -> Swift Packages -> Add Package Dependency` and type in URL address of this repository.
+```
+.package(url: "https://github.com/futuredapp/CellKit", from: "0.8.0")
+```
+### CocoaPods
+Add following line to your  `Podfile` and then run `bundle exec pod install`
+```
+pod 'CellKit', '~> 0.8'
+```
+
+## Usage
+CellKit provides a data source and a section model which you fill with your cells, headers and footer models. All you're left to do is to define your cell view and your cell model with protocol conformance to CellConvertible and CellConfigurable and CellKit will handle the rest.
+
+### 1. step: Set a DataSource
+CellKit provides a `CellModelDataSource` datasource and `GenericCellModelSection` section, which define your `UITableView`/`UICollectionView` cell structure. You can always subclass `CellModelDataSource` and override it's methods to suit your needs.  
+Here's an example of typical CellKit datasource usage:
+```swift
+lazy var dataSource: CellModelDataSource = {
+    CellModelDataSource([
+        GenericCellModelSection(arrayLiteral:
+            PrototypeCellViewModel(name: "Prototype")
+        ),
+        GenericCellModelSection(arrayLiteral:
+            CodeCellViewModel(name: "Code")
+        ),
+        GenericCellModelSection(arrayLiteral:
+            XIBCellViewModel(name: "XIB")
+        )
+    ])
+}()
```

Comment:
I wouldn't confuse the users with the lazy definition. It doesn't make much sense in this particular implementation.

```suggestion
let dataSource = CellModelDataSource([
    GenericCellModelSection(arrayLiteral:
        PrototypeCellViewModel(name: "Prototype")
    ),
    GenericCellModelSection(arrayLiteral:
        CodeCellViewModel(name: "Code")
    ),
    GenericCellModelSection(arrayLiteral:
        XIBCellViewModel(name: "XIB")
    )
])
```